Helper/Quote.php

```diff
@@ -530,13 +528,13 @@ public function getParentSkus()
                 continue;
             } else if($parentItem){
                 $product = $this->productRepository->getById($parentItem->getProduct()->getId());
-                $parentSkus[] = $product->getSku();
+                $parentSkus[$product->getSku()] = null;
             } else {
-                $parentSkus[] = $item->getSku();
+                $parentSkus[$item->getSku()] = null;
             }
         }
 
-        return $parentSkus;
+        return array_keys($parentSkus);
```

Comment:
That's not really a costly operation to do, both setting values to null or array_keys to extract them, and by using sku as array key that makes sure there won't be multiple array members referencing same parent sku if it's used as value, this can happen when iterating over child items.